src/utils/integration/import.ts

```diff
@@ -182,6 +183,17 @@ export const importCodeNativeIntegration = async (integrationId?: string): Promi
   }
   await uploadConnectionIcons(componentDefinition, connectionIconUploadUrls);
 
+  try {
+    const metadata = await getPrismMetadata({ fromDist: true });
+    await writePrismMetadata(
+      { ...metadata, integrationId: integrationImportId },
+      { fromDist: true },
+    );
+  } catch (e) {
+    console.error("Import was successful but there was an error formatting local metadata:");
+    console.error(e);
```

Comment:
```suggestion
    console.error("Import was successful but there was an error formatting local metadata:", e);
```

Was the reason you didn't combine these for formatting reasons?